docs/user/oidc-publishing/index.md

```diff
@@ -0,0 +1,435 @@
+---
+title: About
+---
+
+{{ preview('oidc-publishing') }}
+
+# Publishing to PyPI with OpenID Connect
+
+## Confirming that you're in the beta
+
+Before we do anything else: let's confirm that you're actually in the OIDC
+beta! You can do this by [logging into PyPI](https://pypi.org/account/login/)
+and clicking on "Account settings" in the dropdown under your profile:
+
+{{ image('dropdown.png') }}
```

Comment:
CC @di @ewdurbin I think I'm rendering images incorrectly here -- regardless of whether I use my own macro or `![]` directly with a relative path to the asset, they both 404. Any thoughts?

Reply:
Oh, my bad -- I was trying to use the _host_'s paths for these assets!